### PR TITLE
Set Roll Forward to Major for some tools

### PR DESCRIPTION
**Pull request by sm6srw**
### Purpose

Set Roll Forward to Major for:
* DynamoFeatureFlags.exe
* MD2Html.exe
* DynamoCLI
* DynamoWPFCLI

This works very well on machines where NET7 and not NET6 runtime is installed.

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [ ] This PR contains no files larger than 50 MB

### Release Notes

(FILL ME IN) Brief description of the fix / enhancement. **Mandatory section**

### Reviewers

(FILL ME IN) Reviewer 1 (If possible, assign the Reviewer for the PR)

(FILL ME IN, optional) Any additional notes to reviewers or testers.

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

**Comment by mjkkirschner** [comment]
is this all the .exes we use at runtime?

**Comment by sm6srw** [comment]
> is this all the .exes we use at runtime?

As far as I know, Yes.

The other ones are:

File: src/Tools/Md2Html/Md2Html.csproj [hunk 1]
```diff
@@ -29,6 +30,7 @@
         <CallTarget Targets="Publish" />
         <Exec Command="if not exist $(SharedOutputPath) mkdir $(SharedOutputPath)"/>
         <Copy Condition=" '$(RuntimeIdentifier)' == 'win10-x64' " SourceFiles="$(OutputPath)$(RuntimeIdentifier)\publish\Md2Html.exe" DestinationFolder="$(SharedOutputPath)" />
+        <Copy Condition=" '$(RuntimeIdentifier)' == 'win10-x64' " SourceFiles="$(OutputPath)\Md2Html.runtimeconfig.json" DestinationFolder="$(SharedOutputPath)" />
```
[the remark this answers is not in the document]
Comment (sm6srw) [reply src/Tools/Md2Html/Md2Html.csproj:33]: It is not the root folder. It is the folder where MD2Html.exe ends up.

**Comment by mjkkirschner** [comment]
> When I think about it: We should also add this to DynamoCLI.exe and DynamoWPFCLI.exe.

just those two seem fine to me.

**Review by mjkkirschner — APPROVED** [review]

File: src/Tools/Md2Html/Md2Html.csproj (hunk 1 above)
[the remark this answers is not in the document]
Comment (pinzart90) [reply src/Tools/Md2Html/Md2Html.csproj:33]: We’ll I see this copy step has a destination folder as the sharedOutputfolder

[the remark this answers is not in the document]
Comment (sm6srw) [reply src/Tools/Md2Html/Md2Html.csproj:33]: Yes, this is the one when we pack the exe up with all the dependencies. I could not make it work without also copy this file to the final destination of the exe. There was no runtime config file there before.

[the remark this answers is not in the document]
Comment (pinzart90) [reply src/Tools/Md2Html/Md2Html.csproj:33]: oh, ok. It is the md2html subfolder.

**Review by pinzart90 — APPROVED** [review]

**Pull request merged**